src/NodaTime/Period.cs

```diff
@@ -354,6 +363,16 @@ private Period(long hours, long minutes, long seconds, long milliseconds, long t
                 minuend.Nanoseconds - subtrahend.Nanoseconds);
         }
 
+        /// <summary>
+        /// Subtracts one period from another, by simply subtracting each property value.
+        /// </summary>
+        /// <param name="minuend">The period to subtract the second operand from</param>
+        /// <param name="subtrahend">The period to subtract the first operand from</param>
+        /// <returns>The result of subtracting all the values in the second operand from the values in the first. The
+        /// units of the result will be the union of both periods, even if the subtraction caused some properties to
```

Comment:
On reading this, I wondered if we should say (here and above) that it's valid to e.g. do 2w - 2w1d = -1d (where we treat the first argument as implicitly 2w0d)... but I couldn't find a good way to do so, and maybe it's obvious?